tests/test_lora.py

```diff
@@ -106,3 +109,17 @@ def test_materialise(getkey):
     _ = quax.quaxify(jax.nn.relu)(x_true)
     with pytest.raises(RuntimeError, match="Refusing to materialise"):
         _ = quax.quaxify(jax.nn.relu)(x_false)
+
+
+def test_regression_38():
+    x = jax.numpy.arange(4.0).reshape(2, 2)
+    y = lora.LoraArray(x, rank=1, key=jax.random.key(0))
+
+    def f(x):
+        return jax.lax.add_p.bind(x, y)
+
+    func = quax.quaxify(f)
+
+    # Error type depends on whether jaxtyping is on
+    with pytest.raises((UnexpectedTracerError, TypeCheckError, NotFoundLookupError)):
```

Comment:
Why an unexpected tracer error? That seems like the kind of error that should never happen.

Reply:
Yeah. #42 is the Issue. I'll fix this when that is closed.

Reply:
Done.